.github/workflows/main.yml

```diff
@@ -0,0 +1,173 @@
+name: Nightly Feathr Tests on Windows
+
+on:
+  schedule:
+    - cron: 15 14 * * *
```

Comment:
Better add a comment here explaining what cron expression is resolved to.

Reply:
Good comment, will do.

Reply:
@blrchen  @xiaoyongzhu DO you know why this file won't show up under GH actions workflows? 
<img width="1284" alt="image" src="https://user-images.githubusercontent.com/8405222/169699020-3167ddb9-e5e7-4081-b882-042d1426b52c.png">